tests/xbot/common/properties/PropertyTest.java

```diff
@@ -120,4 +125,12 @@ public void testLoadingValueAfterCreation() {
         assertEquals(true,bool.get());
         assertEquals("teststring",str.get());
     }
+    
+    @After
+    public void cleanUp()
+    {
+    	// We need a way to obliterate the database locally so tests don't leak. Can't delete the files themselves,
+    	// because the database process still has a handle on some of them.
+    	assertEquals(true, ((PersonalComputerDatabaseStorage)propertyManager.permanentStore).obliterateStorage());
```

Comment:
Yeah, it's to make sure that the database is clean for the next test. That way we at least know why other tests may be failing in an unexpected way.